deployer/README.md

```diff
@@ -492,6 +494,34 @@ To set the parameters:
       oc create -n logging -f -
     # this creates the daemonset and starts fluentd with the new params
 
+#### Secure Forward
+
+To be able to accommodate different possible configurations for aggregated logging
+stacks, yet still maintain a light container we are now providing the [secure forward](https://github.com/tagomoris/fluent-plugin-secure-forward)
+plugin to enable customers to forward Fluentd logs from OpenShift's Fluentd to
+an external aggregator which can interpret Fluentd's messages and can then further
+process logs and/or forward them to a log store.
+
+To set up `secure forward`, you need only update the `secure-forward.conf` section of
+the config map provided for you. A sample, commented out, configuration has been
+provided. Further explanation of how to set up `secure forward` can be found
+at it's [repo](https://github.com/tagomoris/fluent-plugin-secure-forward).
+
+  $ oc edit configmap/logging-fluentd
+
+To add certificates to your Fluentd pods to be used in `secure forward` you can
```

Comment:
Certificates can be used with 'secure forward' and Fluentd by updating the secret that is mounted into the pod.